### ISSUE TITLE
How to find hal2liftover from cactus install?

### ISSUE THREAD

**Issue by hopehealey** [ComparativeGenomicsToolkit/cactus#1411]
Hi Cactus team, 

I would like to use the halLiftover tool from the Hal package to transfer phastcons conserved element annotations from one species to another. From the Hal GitHub, it seems as though I should not need to install Hal if I already have cactus in my system. Since I have cactus installed and working properly, I was wondering how to find the halLiftover command within cactus. If I try to run cactus-halLiftover, it does not find the command. I noticed that the hal2chains uses halLiftover, so I know it should be there somewhere. Does anyone have tips to find halLiftover?

Thank you,
Hope

**Comment by glennhickey** [comment]
Yeah, hal's included in the cactus releases in `cactus/bin`.  If you've installed cactus correctly, you should be able to run `halLiftOver` or any other hal tool after activating the cactus environment.
